core/src/components/picker-column/picker-column.scss

```diff
@@ -4,13 +4,42 @@
 // --------------------------------------------------
 
 :host {
-  @include padding(0px, 16px, 0px, 16px);
+  display: flex;
+
+  justify-content: center;
 
   height: 200px;
 
+  font-size: 22px;
+
+  text-align: center;
+}
+
+.picker-opts {
+  /**
+   * This padding must be set here and not on the
+   * host to ensure that the focus highlight on the
+   * column is not overly narrow.
+   */
+  @include padding(0px, 16px, 0px, 16px);
+
+  /**
+   * When using reactive data, such as in datetime's
+   * wheel picker, there may be layout shifts between
+   * individual columns in a picker as the columns
+   * shrink to fit the widest item in the column.
+   * Setting a minimum width avoids this layout shifting.
+   *
+   * Note that we need this min-width set on the same
+   * element as the vertical padding (i.e. not on the
+   * host).
+   */
+  min-width: 26px;
+
   outline: none;
 
-  font-size: 22px;
+  text-align: inherit;
```

Comment:
Datetime adjusts the text alignment of specific columns in its pickers; I added this so we can continue to specify the `text-align` on the host from other components.